src/main/resources/com/parasoft/findings/jenkins/coverage/ParasoftCoverageQualityGate/config_zh_CN.properties

```diff
@@ -1,3 +1,4 @@
 title.threshold=\u9608\u503C
 title.baseline=\u57FA\u7EBF
-title.criticality=\u9636\u6BB5\u6216\u6784\u5EFA\u7ED3\u679C
\ No newline at end of file
+title.criticality=\u9636\u6BB5\u6216\u6784\u5EFA\u7ED3\u679C
+//?????
```

Comment:
The current file was introduced elsewhere and was not written by our internal developers.
Import Link:https://github.com/jenkinsci/parasoft-findings-plugin/commit/ebb5976197b5efda4163d16ab5b4ea007cd2f847

Reply:
This is from a shared user account that checks for differences between GitHub and Aquaman and makes updates. The job runs daily.